client/fingerprint/storage.go

```diff
@@ -53,6 +53,8 @@ func (f *StorageFingerprint) Fingerprint(cfg *config.Config, node *structs.Node)
 	node.Attributes["unique.storage.bytestotal"] = strconv.FormatUint(total, 10)
 	node.Attributes["unique.storage.bytesfree"] = strconv.FormatUint(free, 10)
 
+	f.logger.Printf("[DEBUG] fingerprint.storage: volume: %s, total: %d Byte, free: %d Byte", volume, total, free)
```

Comment:
Since the node attributes can be queried from the cli and api, we don't need a debug message here.